rulebook.tex

```diff
@@ -1630,6 +1639,15 @@ \subsection{Task Fulfillment and Scoring}
 awarded. Performing a step for a later order which has not yet been
 announced cannot be awarded.
 
+For a competitive order, the same scoring scheme for in-time, delayed, and late
+delivery applies, with additional points for a successful first delivery
+and a point deduction for the second delivery (cf.~\reftab{tab:scoring}). The
+deduction may not exceed the points given for the delivery.\footnote{As an
```

Comment:
Place the "." after the footnote. We use the fnpct package which makes the display nicer, but it requires points to be set after the footnote (in LaTeX you seemingly can only do lookahead, but not look back).
```suggestion
deduction may not exceed the points given for the delivery\footnote{As an
```